integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -251,9 +247,9 @@ public void exec(int depth, int concurrencyDepth, TachyonURI path) throws Except
   public ExpectedException mThrown = ExpectedException.none();
 
   @Test
-  public void addCheckpointTest() throws FileDoesNotExistException, SuspectedFileSizeException,
-      FileAlreadyExistException, InvalidPathException, BlockInfoException, FileNotFoundException,
-      TachyonException {
+  public void addCheckpointTest()
```

Comment:
FYI, I think the throws clause could just be Exception here to avoid having to update this test every time a new exception introduced. No need to change it, I have a pending PR that does that.